src/Sugar.mli

```diff
@@ -97,9 +97,9 @@ val ite :
     if-then-else expressions, e.g.:
 
     {[
-        if c1 then e1
-        else if c2 then e2
-        else e3
+if c1 then e1
+else if c2 then e2
+else e3
```

Comment:
Is this indentation intended?

Reply:
A bit of work is needed to fix this. It is intended for verbatim blocks to avoid pushing ascii tables to the margin.

Reply:
Ok. I don't really know what the best choice ought to be, but expect that people will want to be able to keep things out of the left margin. But maybe not  if that means having inexplicable extra space in the generated docs.

Reply:
Whitespaces are preserved in verbatim/code blocks. The user is free to indent it.